infrastructure/src/main/java/org/corfudb/infrastructure/NettyServerRouter.java

```diff
@@ -100,16 +102,26 @@ protected static void handleUncaughtException(Thread t, @Nonnull Throwable e) {
     @Getter
     final List<AbstractServer> servers;
 
+    private final ServerContext serverContext;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "serverContext" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)